tests/ignite/metrics/test_loss.py

```diff
@@ -78,34 +78,17 @@ def test_zero_div():
         loss.compute()
 
 
-def test_compute():
-    def _test(y_test_1, y_test_2):
-        loss = Loss(nll_loss)
+@pytest.mark.parametrize("criterion", [(nll_loss, nn.NLLLoss())])
```

Comment:
Wrong parametrization here
```suggestion
@pytest.mark.parametrize("criterion", [nll_loss, nn.NLLLoss()])
```